library/tomee

```diff
@@ -19,4 +19,8 @@
 8-jdk-7.0.1-plus: git://github.com/tomitribe/docker-tomee@8c5b5a96305a9b935d304789fcf3707938a175d6 8-jdk-7.0.1-plus
 8-jdk-7.0.1-webprofile: git://github.com/tomitribe/docker-tomee@8c5b5a96305a9b935d304789fcf3707938a175d6 8-jdk-7.0.1-webprofile
 
-latest: git://github.com/tomitribe/docker-tomee@8c5b5a96305a9b935d304789fcf3707938a175d6 8-jdk-7.0.1-webprofile
+8-jre-7.0.3-plume: git://github.com/tomitribe/docker-tomee@b8f51bbf3eb284101b404a5eb1f412f1cfb9346b 8-jre-7.0.3-plume
+8-jre-7.0.3-plus: git://github.com/tomitribe/docker-tomee@bb8f51bbf3eb284101b404a5eb1f412f1cfb9346b 8-jre-7.0.3-plus
```

Comment:
Might be worth considering the new RFC 2822-based format to help prevent these sorts of problems (https://github.com/docker-library/official-images/tree/5c861ba1f8e852bf83d6e19c493ca962999fec28#instruction-format), ala:

```
Maintainers: Alex Soto <asotobu@gmail.com> (@lordofthejars)
GitRepo: https://github.com/tomitribe/docker-tomee.git
GitCommit: b8f51bbf3eb284101b404a5eb1f412f1cfb9346b

Tags: 6-jre-1.7.4-jaxrs
Directory: 6-jre-1.7.4-jaxrs

Tags: 6-jre-1.7.4-plume
Directory: 6-jre-1.7.4-plume

Tags: 6-jre-1.7.4-plus
Directory: 6-jre-1.7.4-plus

Tags: 6-jre-1.7.4-webprofile
Directory: 6-jre-1.7.4-webprofile

Tags: 7-jre-1.7.4-jaxrs
Directory: 7-jre-1.7.4-jaxrs

Tags: 7-jre-1.7.4-plume
Directory: 7-jre-1.7.4-plume

Tags: 7-jre-1.7.4-plus
Directory: 7-jre-1.7.4-plus

Tags: 7-jre-1.7.4-webprofile
Directory: 7-jre-1.7.4-webprofile

Tags: 8-jre-1.7.4-jaxrs
Directory: 8-jre-1.7.4-jaxrs

Tags: 8-jre-1.7.4-plume
Directory: 8-jre-1.7.4-plume

Tags: 8-jre-1.7.4-plus
Directory: 8-jre-1.7.4-plus

Tags: 8-jre-1.7.4-webprofile
Directory: 8-jre-1.7.4-webprofile

Tags: 8-jdk-7.0.1-plume
Directory: 8-jdk-7.0.1-plume

Tags: 8-jdk-7.0.1-plus
Directory: 8-jdk-7.0.1-plus

Tags: 8-jdk-7.0.1-webprofile, latest
Directory: 8-jdk-7.0.1-webprofile
```